src/components/BaseTextInput/index.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {
+    // eslint-disable-next-line no-restricted-imports
+    TextInput as RNTextInput,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+
+const propTypes = {
+    /** A ref to forward to the text input */
+    forwardedRef: PropTypes.func,
+};
+
+const defaultProps = {
+    forwardedRef: null,
+};
+
+class BaseTextInput extends Component {
+    render() {
+        return (
+            <RNTextInput
+                // eslint-disable-next-line rulesdir/prefer-early-return
+                ref={(ref) => {
+                    if (typeof this.props.forwardedRef === 'function') { this.props.forwardedRef(ref); }
+                }}
+
+                // By default, align input to the left to override right alignment in RTL mode which is not yet supported in the App.
+                // eslint-disable-next-line react/jsx-props-no-multi-spaces
+                textAlign="left"
+
+                // eslint-disable-next-line
+                {...this.props}
+            />
+        );
+    }
+}
```

Comment:
Please convert this to a class component...

Reply:
I don't get this, it's a class component.

Reply:
also, take look here @parasharrajat

Reply:
Sorry, mistype... I mean functional component. :small_airplane: